<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <rosparam command="load" file="$(find mm_apriltags_tracker)/launch/param/pos.yaml" />
  <node name="camera1" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video1" />
    <param name="image_width" value="960" />
    <param name="image_height" value="720" />
    <param name="pixel_format" value="mjpeg" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>

  <node name="image_view1" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/camera1/image_raw"/>
    <param name="autosize" value="true" />
  </node>

   <node name="mm_april_tags_tracker1" pkg="mm_apriltags_tracker" type="mm_april_tags_tracker"  output="screen">
     <remap from="image" to="/camera1/image_raw"/>
   </node>
</launch>
